### ISSUE TITLE
Emscripten - A input debug popup box opens up.

### ISSUE THREAD

**Issue by Pakz001** [blitz-research/monkey2#129]
I have not taken apart and pinpointed the code what causes it yet but maybe it is a easy fix. The desktop compile works but with emscripten I get that input popup box showing up a few seconds after the program gets active.  Might be in that I am using createimage and canvasses and drawing in them. 

I am using the binary release version.

**Comment by chriscamacho** [comment]
just a thought... but if you posted the *minimum* code example that will make this happen for you, then it is likely to save a lot of time, or even actually make it possible to help you...

**Comment by Pakz001** [comment]
I was still looking into this. So far I did not completely strip the code to find it. I might make some time later tonight and pinpoint it.  The bug with the Double variables took me a while to find to.

I initially though maybe mark would know how to ctrl+f the input box.

**Comment by Pakz001** [comment]
I just checked again and now the popup does not show itself. The code is the same. On latest chrome. on win 10 64bit. When I get it again I will post a follow up.

**Comment by blitz-research** [comment]
Can you remember what the popup even said?

**Comment by Pakz001** [comment]
Only that it said something about inputting a debug value. It was a small popup window that seemed to come from the emscripten. It was a week ago or so. It happened every time I ran it after a few seconds it appeared.
I tried compiling other code to emscripten and there it did not happen. The file was rather big so it would of taken me a while to pinpoint.

If it is any help it was this code : (big one sorry)
https://github.com/Pakz001/Monkey2examples/blob/master/2d%20Platformer%20RPG%2003.monkey2

**Comment by Pakz001** [comment]
I am sorry. I just found out after getting that error again that it is caused by that bug report I made earlier.

I had used a couple of doubles and declared before that double was a image. The overwrite causes the image to be modified and that popup box being displayed. 
So :
field image:Image
field rot:Double <<< overwrites image if modified in code

What maybe is also worth mentioning is that I also modified the config file to get more memory for I got those out of memory (arrays) messages. Not tested that and the double error,
